### ISSUE TITLE
Top Loci table

### ISSUE THREAD

**Issue by mfazel** [statgen/locuszoom-hosted#33]
Hi,

I use locusZoom occasionally to brows some GWAS data and create plots, which I'd say, they are nicely generated compare to amount of time a user spends. This is just a suggestion not a big problem.
Looking at the Loci table right bellow the Manhattan plot in the study page. By default it's sorted by Pvalue, which is fine, (and I assume the table presentation is done in java script) but there is no search option for the table, not a general search nor a column by column search box. The number of rows in each page is also fixed on 10 and it's not possible to extend the table to more rows to see more rows easily.
The problem is if a study has 20 different pages (10 row each), and user wants to go to row number 171, has to go through all pages until that row and then if I go to that locus and then come back, I'll land on the original table sorted by Pvalue (unless open it in a new tab).

Thanks for your nice tool.

**Comment by abought** [comment]
Thanks for the question! We can definitely look at improving the top loci table in the future.

In the interim, you may find it convenient to know that the manhattan plot shows much of the same information as the table, and that you an also navigate to loci by clicking directly on significant points in the manhattan plot.

**Comment by abought** [comment]
Looking back at this proposal in more detail: is it correct to say that you want a way to find a particular variant, and that you want to browse all hits in sequence?

(eg, you want to go to row 171 because it is the next hit- the position in the table isn't otherwise important as long as you can find an entity of interest)

I wonder if two navigation features in the "region" page would help solve your problem better?
1. Rather than clicking hits in order from the "top hits" table, the "batch view" button lets you fetch that same list of interesting loci, and step through each hit one by one
2. The "go to region" search box accepts a gene name, rsID, or `chr:pos` specifier to jump to a specific region of interest.

Can you offer an example of where searching the table would be preferable to looking for your locus in either the manhattan plot, or via the region page?

**Comment by abought** [comment]
I see. By default batch view sorts "top hits first". 

You'd like to see that combined with a filter based around the range of positions? We've been tossing around some concepts for this sort of thing internally; I'll give some thought to this, though we might have to pilot the concept in some other pending work first.

Action items:
- [ ] Allow user to filter top loci table by range of positions (supported formats: _all in chrom_, or _c:b-e_)
- [ ] Show more items in that table by default
- [ ] Provide an "export table" button on top loci page so that users can explore and slice top hits in any manner of their choosing

**Comment by mfazel** [comment]
Yes, something like that, but again it doesn't need to be fancy and I think a couple of features of java script table views are very helpful and you see them almost in every website. (I'm guessing this is also done in java script!)
Here is an example link (and just in case I include the screenshot too). 
You see there is an adjustable number of rows by user (10,25,50.....) on top left, a general search on top right and download link below the table and user can search for anything not only coordinates. Some other websites also include searching on each column header separately to allow for "AND" operation.
Thanks,

https://ldlink.nci.nih.gov/?var=rs34330&pop=CHB%2BJPT%2BCHS%2BCDX%2BKHV&genome_build=grch37&r2_d=r2&window=500000&collapseTranscript=true&tab=ldproxy


![image](https://user-images.githubusercontent.com/13437418/171222771-513d04d2-9fb5-41a1-9658-8e6c8fcbb974.png)
